README.md

```diff
@@ -4,9 +4,45 @@ The python data pipeline defined with DataJoint for U19 projects
 
 The data pipeline is mainly ingested and maintained with the matlab repository: https://github.com/shenshan/U19-pipeline-matlab
 
-This repository is the mirrored table definitions for the tables.
+This repository is the mirrored table definitions for the tables in the matlab pipeline.
 
-## Major schemas
+# Installation
+
+## Prerequisites (for recommended conda installation)
+
+1. Install conda on your system:  https://conda.io/projects/conda/en/latest/user-guide/install/index.html
+2. If running in Windows get [git](https://gitforwindows.org/)
+3. (Optional for ERDs) [Install graphviz](https://graphviz.org/download/)
+
+## Installation with conda
+
+1. Open a new terminal 
+2. Clone this repository: `git@github.com:BrainCOGS/U19-pipeline_python.git`
+    - If you cannot clone repositories with ssh, [set keys](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)
+3. Create a conda environment: `conda create -n u19_datajoint_env python==3.7`.
```

Comment:
Should we change the python version to 3.8.11, as 3.7 may be the reason that there was an import error with `paramiko`, we can definitely test this a little more before making the change.

Reply:
Resolving this here, will update the version in the next PR.